src/Zypper.cc

```diff
@@ -698,6 +698,7 @@ void print_main_help( Zypper & zypper )
 
   static std::string help_global_target_options = _("     Target Options:\n"
     "\t--root, -R <dir>\tOperate on a different root directory.\n"
+    "\t--installroot <dir>\tOperate on a different root directory, but share repositories with the host.\n"
     "\t--disable-system-resolvables\n"
     "\t\t\t\tDo not read installed packages.\n"
```

Comment:
Like with the command options, we should not change the existing translated string. For now I'd append the option:
```
help_global_target_options += str::Format( "\t--installroot <dir>\t%1%\n" ) % _(Operate on a different root directory, but share repositories with the host.)
```

Reply:
Sorry i keep forgetting that

Reply:
Done